config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
```

Comment:
pages#home = PagesController, Action Home
We do not have a Pages controller so it should be DronesController (drones#home)

Reply:
You are saying n'importe quoi Andy, look in your app/controllers file ;)

Reply:
desoleeeeeeeee